src/components/filters/Cuisine.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { fetchRestaurants } from '../../actions';
+import styles from '../Filters.module.sass'
+
+class Cuisine extends React.Component {
+    state = { cuisines: [] };
+
+    componentWillReceiveProps(nextProps) {
```

Comment:
You shouldn't use `componentWillReceiveProps` as it will be deprecated in React v17 due to the fact that this lifecycle method often leads to bugs and inconsistencies. One reason is because sometimes it’s called when nothing has changed; React just wants to check in.
[see also](https://reactjs.org/docs/react-component.html). Anyway, you should consider instead using `componentDidUpdate` (invoked only after update occurs, i.e when received new props or state) or if possible on 'componentDidMount'